tests/models/hf_equivalence/test_gpt_bigcode.py

```diff
@@ -0,0 +1,129 @@
+import pytest
+
+from fms.models.gpt_bigcode import GPTBigCode
+from fms.models.hf.gpt_bigcode.modeling_gpt_bigcode_hf import (
+    HFAdaptedGPTBigCodeForCausalLM,
+)
+from fms.testing.comparison import (
+    ModelSignatureParams,
+    HFModelSignatureParams,
+    compare_model_signatures,
+)
+
+
+@pytest.mark.slow
+def test_gptbigcode_equivalence():
+    """Tests GPT BigCode equivalence with a known implementation. Takes approximately 1:11 on an mbp with M1 chip"""
+    import torch
+    from transformers import AutoModelForCausalLM, AutoTokenizer, pipeline
+
+    tokenizer = AutoTokenizer.from_pretrained("bigcode/gpt_bigcode-santacoder")
+    hf_model = AutoModelForCausalLM.from_pretrained("bigcode/gpt_bigcode-santacoder")
+    hf_model.config.scale_attention_softmax_in_fp32 = False
+
+    model = GPTBigCode(
```

Comment:
can you reuse the `get_model()` function in `fms/models/hf/gpt_bigcode/__init__.py`?

Reply:
adding to latest commit